multiuser/integration-tests/che-multiuser-mysql-tck/pom.xml

```diff
@@ -252,11 +252,7 @@
                                             <port>jdbc.port:3306</port>
                                         </ports>
                                         <wait>
-                                            <tcp>
-                                                <ports>
-                                                    <port>3306</port>
```

Comment:
Is it cleaning up or changes related to Stack API?

Reply:
synced with Postgres. Without that, I can't test it on OSX